client/src/pages/listings/Listings.jsx

```diff
@@ -11,13 +11,8 @@ const Listings = () => {
 
   const getLeaseData = () => {
     let queryUrl = "?id=" + window.sessionStorage.getItem("userId");
-    let headers = {
-      'Content-Type': 'application/json',
-      "Access-Control-Allow-Origin": "*"
-    };
 
-    fetch(process.env.REACT_APP_SERVER_URL + "list" + queryUrl,
-      { headers })
+    fetch(process.env.REACT_APP_SERVER_URL + "list" + queryUrl)
```

Comment:
are these headers no long needed?

Reply:
We don't need them anymore since CORS issues are resolved in the backend.